admin/src/Components/Login.jsx

```diff
@@ -31,14 +31,14 @@ export default class Login extends React.Component
                         <div className="uk-form-row">
                             <div className="uk-form-icon">
                                 <i className="uk-icon-user"/>
-                                <input ref="username" className="uk-form-large uk-form-width-large" type="text" placeholder="Användarnamn" />
+                                <input ref={c => { this.username = c; }} className="uk-form-large uk-form-width-large" type="text" placeholder="Användarnamn" />
                             </div>
                         </div>
 
                         <div className="uk-form-row">
                             <div className="uk-form-icon">
                                 <i className="uk-icon-lock"/>
-                                <input ref="password" className="uk-form-large uk-form-width-large" type="password" placeholder="Lösenord" />
+                                <input ref={c => { this.password = c; }} className="uk-form-large uk-form-width-large" type="password" placeholder="Lösenord" />
```

Comment:
It's probably a good idea to drop the ref thing altogether and just use a controlled component (by adding state somewhere. Also, hooks are now available)

Reply:
Yes, I agree.